src/app/cli/README.md

```diff
@@ -205,33 +205,105 @@ fdaa:bb::de6
 
 ### Steering dataset
 
-The Commissioner steers joining of new devices by `Steering Data` in the Commissioner dataset. To add a new CCM AE Joiner:
+The Commissioner steers joining of new devices by `Steering Data` in the Commissioner dataset.
 
 ```shell
+> help joiner
 usage:
-joiner enable (meshcop|ae|nmkp) <joiner-eui64>
-joiner enableall (meshcop|ae|nmkp)
+joiner enable (meshcop|ae|nmkp) <joiner-eui64> [<joiner-password>] [<provisioning-url>]
+joiner enableall (meshcop|ae|nmkp) [<joiner-password>] [<provisioning-url>]
 joiner disable (meshcop|ae|nmkp) <joiner-eui64>
 joiner disableall (meshcop|ae|nmkp)
 joiner getport (meshcop|ae|nmkp)
 joiner setport (meshcop|ae|nmkp) <joiner-udp-port>
 [done]
 >
-
-### The second argument of the joiner command is the type of joiner.
-### It can be only ae, nmkp, or meshcop.
-> joiner enable ae 0123456789abcdef
-[done]
->
 ```
 
-To enable joining for all Joiners:
-
-```shell
-> joiner enableall ae
-[done]
->
-```
+* to enable a new MeshCoP joiner:
+
+    ```shell
+    ### The second argument of the joiner command is the type of joiner.
+    ### It can be only ae, nmkp, or meshcop.
+    > joiner enable meshcop 0x0123456789abcdef JOINER1
```

Comment:
"O" is not a valid character. The Joining Device Credential is composed of ASCII characters drawn from an alphabet of 32 numbers and letters [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, A, B, C, D, E, F, G, H, J, K, L, M, N, P, R, S, T, U, V, W, X, Y].

Reply:
We lack the validation here, I will add this soon.

Reply:
At least update the doc to not use invalid characters.

Reply:
Yes, updated.